cli/flox/doc/flox-activate.md

```diff
@@ -93,11 +105,6 @@ See [`manifest.toml(5)`](./manifest.toml.md) for more details on shell hooks.
 
 ## Variables set by `flox activate`
 
-`$FLOX_ENV`
-:   Contains the path to the built environment. This directory contains a merged
-    set of `bin`, `lib`, etc directories for all the packages in the
-    environment.
-
```

Comment:
(blocking) We lost 8fa609a2d54a7f8c563cda5aa87f9816f352d02f in copying back the file that was forked at an older version.

Reply:
Fixed